script/constructron.lua

```diff
@@ -799,46 +727,76 @@ me.conditions = {
     ---@param position MapPosition
     ---@return boolean
     position_done = function(job, position) -- this is condition for action "go_to_position"

```

Comment:
the return value annotation seems to be incorrect now (and was actually incorrect before lol).
```suggestion
    ---@param position MapPosition
    ---@return boolean?
    position_done = function(job, position) -- this is condition for action "go_to_position"
```